app/views/idv/by_mail/letter_enqueued/show.html.erb

```diff
@@ -4,47 +4,40 @@
         current_step: :get_a_letter,
         locale_scope: 'idv',
         class: 'margin-x-neg-2 margin-top-neg-4 tablet:margin-x-neg-6 tablet:margin-top-neg-4',
-      ) %>
+      )
+  %>
 <% end %>
 
 <% self.title = t('titles.idv.come_back_soon') %>
 
 <%= image_tag(
       asset_url('come-back.svg'),
-      size: '140', class: 'display-block margin-x-auto margin-bottom-2',
+      size: '140', class: 'display-block margin-left-0 margin-bottom-2',
       alt: t('idv.images.come_back_later')
-    ) %>
+    )
+%>
 
-<%= render PageHeadingComponent.new(class: 'text-center').with_content(t('idv.titles.come_back_later')) %>
+<%= render PageHeadingComponent.new(class: 'text-left').with_content(t('idv.titles.come_back_later')) %>
 
-<div class="text-center">
-  <%= t('idv.messages.come_back_later_html', app_name: APP_NAME) %>
-</div>
+<p><%= t('idv.gpo.will_send_to') %></p>
 
-<div class="padding-4 text-center border rounded-xl">
-  <p>
-    <%= t('idv.messages.come_back_later_password_html') %>
-  </p>
-  <p>
-    <% if decorated_sp_session.sp_name.present? %>
-      <%= t('idv.messages.come_back_later_sp_html', sp: decorated_sp_session.sp_name) %>
-    <% else %>
-      <%= t('idv.messages.come_back_later_no_sp_html', app_name: APP_NAME) %>
+<div class="grid-row grid-gap grid-gap-2 padding-bottom-1 padding-top-1">
+  <dl class='grid-col-fill margin-y-0'>
+    <% @presenter.address_lines.each do |address_line| %>
+      <div>
+        <dd class="margin-left-0"> <%= address_line %> </dd>
```

Comment:
I'd expect a `<dl>` element to be a set of `<dt>` and `<dd>` pairs, but there's no `<dt>` here. Should we just use simple line breaks instead and avoid the `<dl>` element?

>The element encloses a list of groups of terms (specified using the [`<dt>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dt) element) and descriptions (provided by [`<dd>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dd) elements).

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl